packages/fx-core/templates/plugins/resource/apim/bicep/parameters.json

```diff
@@ -0,0 +1,14 @@
+{
+  "apimClientId": {
+    "value": "{{FX_RESOURCE_APIM__APIMCLIENTAADCLIENTID}}"
```

Comment:
@wenytang-ms is working to change the placeholder style, please work with him to align the placeholder values here

Reply:
This may be fixed when try to merge the code into dev branch.